notebooks/AutoEncoder_Evaluation.ipynb

```diff
@@ -0,0 +1,565 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "###################################################################################################\n",
+    "#\n",
+    "# Copyright (C) 2023 Analog Devices, Inc. All Rights Reserved.\n",
+    "# This software is proprietary and confidential to Analog Devices, Inc. and its licensors.\n",
+    "#\n",
+    "###################################################################################################\n",
+    "import os\n",
```

Comment:
Please add a link to the documentation at the top of the notebook

Reply:
Documentation is currently on pr, link will be added when it is published.

Reply:
We already know the location of the documentation, please add it now so we don't need any further changes

Reply:
Updated